sdk/provisioning/Azure.Provisioning.CosmosDB/src/Generated/CosmosDBSqlRoleAssignment.cs

```diff
@@ -18,11 +18,12 @@ namespace Azure.Provisioning.CosmosDB;
 public partial class CosmosDBSqlRoleAssignment : ProvisionableResource
 {
     /// <summary>
-    /// Gets the Name.
+    /// Gets or sets the Name.
     /// </summary>
     public BicepValue<string> Name 
     {
         get { Initialize(); return _name!; }
+        set { Initialize(); _name!.Assign(value); }
```

Comment:
Note that this is a "Generated" file. See line 4. I don't think we should be hand-editing these files. Instead something is either wrong in the code generator, or in the metadata it is generating the code from.